src/components/CardAddress/Select/index.tsx

```diff
@@ -134,7 +134,7 @@ const CardAddress: React.FC<CardAddressProps> = ({
   )
 }
 
-CardAddress.defaultProps = {
+SelectAddress.defaultProps = {
```

Comment:
I believe that data such as address, address type, post code and complement should not have this type of fallback. If one of them does not exist, incorrect data will be displayed to the user.

Reply:
@samuelvermeuln Use empty string instead of [removing these default properties](https://github.com/vissimo-group/chardonnay/pull/5/commits/15e9daba9cad8aa0a15bee1b800e647801e1ada0)